### DISCUSSION TITLE
is it possible to create static application files?

### DISCUSSION THREAD

**Discussion by MrSuricate2** [symfony/symfony#54836] [Q&A] [answered]
is it possible to create static application files ? I would like to create static files in order to deploy it on my production server. A bit like react can do.

**Comment by MatTheCat** [comment, marked as answer]
I’ve never done it but I guess you can create a [PHAR](https://en.wikipedia.org/wiki/PHAR_(file_format)) from your application.

Note that FrankenPHP seems to be able to include a webserver, PHP and your application in a single executable: https://frankenphp.dev/docs/embed/
